drf_user/views.py

```diff
@@ -441,3 +441,38 @@ def post(self, request, *args, **kwargs):
                 content="Password Updated Successfully.",
                 status=status.HTTP_202_ACCEPTED,
             )
+
+
+class UploadImageView(APIView):
+    """This API can be used to upload a profile picture for user.
+
+    usage: Create a multipart request to this API, with your image
+    attached to `profile_image` parameter.
+    """
+
+    from .models import User
+    from .serializers import ImageSerializer
+    from rest_framework.permissions import IsAuthenticated
+    from rest_framework.parsers import MultiPartParser
```

Comment:
@iamvivekkaushik Please move these imports to top level and use absolute imports.